server/scrape_negative

```diff
@@ -10,15 +10,19 @@ from definitions import NEGATIVE_QUERIES
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(prog="scraper")
+    parser.add_argument("-n", "--narticles", type=int, action="store", default=0)
     args = parser.parse_args()
     from database.mock_database import DatabaseNegative
 
     for query in NEGATIVE_QUERIES:
-        urls = GoogleScraper.find_news_urls_for_query(query)
+        print("scraping for query [{}]".format(query))
+        urls = GoogleScraper.find_news_urls_for_query(query, args.narticles)
```

Comment:
running `scrape_negative` with no arguments never terminates since it calls `GoogleScraper.find_news_urls_for_query(<query>, None)`